tests/test_model.py

```diff
@@ -522,6 +522,48 @@ def test_highlevel_predict_with_invalid_pred_params(self):
         with self.assertRaises(AttributeError):
             model.predict(task, X_t=self.da, pred_params=["invalid_param"])
 
+    def test_patchwise_prediction(self):
+        """Test that ``.predict_patch`` runs correctly."""
+
+        patch_size = (0.6, 0.6)
```

Comment:
Sure, this shouldn't really be relevant to the test since it's only really checking that the code runs and not the quality of the output.

I tried the settings you suggest in this test (fed3940) and it causes an error, on this branch at least

```
ValueError: Resulting object does not have monotonic global indexes along dimension x1
```

Is this something that has been fixed elsewhere?